File: javascript/Q13.js
// 13. Check if the array arr = [1, 2, 3, 4, 5] contains the number 3 and print true or false.

let arr=[1,2,3,4,5];
let flag;
let len=arr.length;
for(i=0;i<len;i++){
    if(arr[i]==3){
   flag=1;
    }
}
if(flag==1){
    console.log("true")
}else{
    console.log("false");
}

File: c lang/Q13.c
// 13. Check if the array arr = [1, 2, 3, 4, 5] contains the number 3 and print true or false
# include <stdio.h>
int main(){
    int arr[]={1,2,3,4,5};
    int len=sizeof(arr)/sizeof(arr[0]);
    int i;
    int flag=0;
    for(i=0;i<len;i++){
        if(arr[i]==3){
        flag=1;
        }
    }
if(flag==1){
    printf("true");
}else{
    printf("false");

  }  

}
